iron-page-url.html

```diff
@@ -272,7 +272,11 @@
 
       // It only makes sense for us to intercept same-origin navigations.
       // pushState/replaceState don't work with cross-origin links.
-      var url = new URL(href, document.baseURI);
+      var url = new URL(href);
```

Comment:
Maybe only assign the value based on the condition, so that you don't end up creating two instances when `document.baseURI != null`?